### DISCUSSION TITLE
RSS for tracking version changes

### DISCUSSION THREAD

**Discussion by brainwo** [jsr-io/jsr#559] [Ideas]
Is there an RSS feed available for tracking version changes of a package? I need some ways to get notification on package updates. I find having RSS is very helpful in other package repositories I use.

Examples of package repository with RSS:
https://pypi.org/rss/project/pillow/releases.xml
https://www.nuget.org/packages/Newtonsoft.Json/atom.xml
https://rubygems.org/gems/bundler/versions.atom

**Comment by SrGeneroso** [comment]
I think this could be also include with this other feature ([Follow/star packages](https://github.com/jsr-io/jsr/discussions/792)). Maybe is not a RSS feed but it could show if the packages you follow have updates or new releases.

**Comment by AugustinMauroy** [comment]
Tracked in #1030
